src/index.js

```diff
@@ -0,0 +1,130 @@
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
+
+const state = {
+    temperature: 40
+};
+
+const tempText = document.querySelector("#tempValue");
+const landscapeText = document.querySelector("#landscape");
+const cityText = document.querySelector("#cityNameInput");
+const skyText = document.querySelector("#sky");
+
+const changeSky = (sky) => {
+    if (sky === "sunny") {
+        skyText.innerHTML = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️"
+    } else if (sky === "cloudy") {
+        skyText.innerHTML = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️"
+    } else if (sky === "rainy") {
+        skyText.innerHTML = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧"
+    } else if (sky === "snowy") {
+        skyText.innerHTML = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+    }
+};
+
+const changeSkyValue = () => {
+    const skyValue = document.getElementById('skySelect').value;
+    changeSky(skyValue);
+};
+
+const increaseTemp = () => {
+    state.temperature += 1;
+    updateTempUi();
+};
```

Comment:
Notice that this method does essentially the same thing as decreaseTemp but the difference is -= 1 or += 1 (also in JS you can use ++ or -- which is short hand for doing the same thing).

How could you combine the logic of these two methods into one event handler called handleTempChange()?

What if you pass in the event object as an argument and you check if the event's target has an id `increaseTempControl` or `decreaseTempControl` and depending on the id then you could ++ or --

```javascript
const handleTempChange = (event) => {
    if (event.target.id === 'increaseTempControl') {
        state.temperature++;
    } else {
        state.temperature--;
    }
    updateTempUi();
};